product_links/README.rst

```diff
@@ -0,0 +1,36 @@
+a module adds links between products
+===========================================
```

Comment:
Should be "Product Links" and the `=` line should be of the sale length (otherwise the rst parser raises an error)